content/geometry/object-vehicle/vehicle-structure.adoc

```diff
@@ -1614,15 +1612,15 @@ NOTE: Add a figure.
 |===
 
 
-== Grp_Mirror_Rearview_Joint_Left_<rearview_mirror_joint_idx>
+== Grp_Mirror_Rearview_Joint_<rearview_mirror_joint_idx>
 
```

Comment:
I noticed a typo in the section title and removed the "_Left".

@LudwigFriedmann: Please doublecheck whether that change was correct.

Reply:
This is indeed a typo. I thought I fixed this already. Hm maybe not. Thanks!